packages/language/languages/en.ts

```diff
@@ -404,7 +404,9 @@ export default {
   payPalLanguage: {
     connectPayPal: `Connect PayPal`,
     connectSuccess: `PayPal enabled successfully.`,
+    disconnectPayPal: `Disconnect PayPal`,
     disconnected: `PayPal disconnected.`,
+    confirm: `CONFIRM`,
```

Comment:
rather than adding this 'CONFIRM', use confirmAccount from language to avoid adding extra translations

Reply:
Thanks, I saw that we have 'CONFIRM' repeated as a language item at least 2-3 times, I wasn't sure which one to pick.